lib/manifestTools/assets/web-manifest.json

```diff
@@ -37,7 +37,12 @@
       "description": "Represents the URL that the developer would prefer the user agent load when the user launches the web application.",
       "type": "string",
       "format": "uri"
+    },
+    "theme_color": {
+      "description": "background color for splash screen and icons",

```

Comment:
yea we really are targeting splash background color and tile background color here.  On windows they can actually be different as well.  Maybe in the next version we'll do a mjs_ version of colors on the tiles and splash directly and see how it goes.
  I never picked up on the idea in the spec, but that shouldn't be hard to implement (now a feature windows / edge does today).  Likewise for the chrome, you can change the color with js, but would be nice to be able to set the color ahead of js execution for a better user experience.